### PR TITLE
add header + mobile

### PR DESCRIPTION
**Pull request by Sysqwerty**

### CODE REVIEW

File: src/js/mobile-menu.js [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
+  mobileMenu.classList.toggle('is-open');
+  const scrollLockMethod = !isMenuOpen
+    ? 'disableBodyScroll'
+    : 'enableBodyScroll';
+  bodyScrollLock[scrollLockMethod](document.body);
+};
+openMenuBtn.addEventListener('click', toggleMenu);
+closeMenuBtn.addEventListener('click', toggleMenu);
+menuLinks.forEach(link => {
+  link.addEventListener('click', () => {
```
Comment (kazamov) [inline src/js/mobile-menu.js:18]: Simplify to `link.addEventListener('click', toggleMenu);`

File: src/partials/mobile_menu.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
+<div class="menu-backdrop js-menu-container">
+  <div class="menu-container" id="mobile-menu">
+    <button class="menu-toggle menu-close-btn js-close-menu">
+      <svg class="menu-close-icon" width="32" height="32">
+        <!-- <use href="../images/icons.svg#icon-close"></use> -->
```
Comment (kazamov) [inline src/partials/mobile_menu.html:5]: Remove commented code

**Review by kazamov — CHANGES_REQUESTED** [review]

File: src/css/header.css [hunk 3]
```diff
@@ -1,3 +1,124 @@
...
+
+.menu-soc-list {
+  gap: 12px;
+}
+
+.menu-soc-link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 44px;
```
Comment (kazamov) [inline src/css/header.css:78]: Do not set width and height. Instead add `padding: 7px`

File: src/css/header.css [hunk 4]
```diff
@@ -1,3 +1,124 @@
 .header {
```
Comment (kazamov) [inline src/css/header.css:1]: Set `outline: transparent` for the social links and navigation links

File: src/css/header.css [hunk 5]
```diff
@@ -1,3 +1,124 @@
...
+  .menu-soc-list {
+    display: flex;
+  }
+}
+
+.nav-list {
+  gap: 20px;
+}
+
+.nav-link {
```
Comment (kazamov) [inline src/css/header.css:45]: Add `padding: 13px 0;` to make an clickable area bigger

File: src/css/header.css [hunk 6]
```diff
@@ -1,3 +1,124 @@
...
+}
+
+.nav-link::after,
+.nav-link::after {
+  content: '';
+  display: block;
+  width: 37px;
+  height: 2px;
+  position: absolute;
+  bottom: -4px;
```
Comment (kazamov) [inline src/css/header.css:59]: Change bottom position to `9px`

**Review by kazamov — CHANGES_REQUESTED** [review]

File: src/css/common.css [hunk 7]
```diff
@@ -0,0 +1,57 @@
+.section {
```
[the remark this answers is not in the document]
Comment (Sysqwerty) [reply src/css/common.css]: Done

File: src/css/common.css [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+  padding: 40px 0;
+}
+
+.container {
+  max-width: 375px;
```
[the remark this answers is not in the document]
Comment (Sysqwerty) [reply src/css/common.css]: Done

File: src/css/common.css [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+.main-title {
+  font-weight: 700;
+  font-size: 38px;
+  line-height: 1.05263;
+  color: rgb(var(--white));
+}
+
+.main-title-span {
```
[the remark this answers is not in the document]
Comment (Sysqwerty) [reply src/css/common.css]: Done

File: src/css/common.css [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+  padding: 40px 0;
+}
+
+.container {
+  max-width: 375px;
+  margin: 0 auto;
+  padding: 0 20px;
+}
+
+@media screen and (min-width: 768px) {
```
[the remark this answers is not in the document]
Comment (Sysqwerty) [reply src/css/common.css]: Done

File: src/css/common.css [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+  .container {
+    max-width: 768px;
+    padding: 0 32px;
+  }
+}
+
+@media screen and (min-width: 1280px) {
```
[the remark this answers is not in the document]
Comment (Sysqwerty) [reply src/css/common.css]: Done

File: src/css/main.css [hunk 12]
```diff
@@ -1,3 +1,13 @@
-@import url('./base.css');
-@import url('./header.css');
-@import url('./footer.css');
+@import url('variables.css');
+@import url('base.css');
+@import url('common.css');
```
[the remark this answers is not in the document]
Comment (Sysqwerty) [reply src/css/main.css]: Done

File: src/css/variables.css [hunk 13]
```diff
@@ -0,0 +1,18 @@
+:root {
```
[the remark this answers is not in the document]
Comment (Sysqwerty) [reply src/css/variables.css]: Done

File: src/images/icons.svg
[the remark this answers is not in the document]
Comment (Sysqwerty) [reply src/images/icons.svg:1]: Done

File: src/js/mobile-menu.js (hunk 1 above)
Comment (Sysqwerty) [reply src/js/mobile-menu.js]: Done

File: src/partials/mobile_menu.html (hunk 2 above)
Comment (Sysqwerty) [reply src/partials/mobile_menu.html]: Done

File: src/css/header.css (hunk 3 above)
Comment (Sysqwerty) [reply src/css/header.css:80]: Preffer to left as is

File: src/css/header.css (hunk 4 above)
Comment (Sysqwerty) [reply src/css/header.css:1]: Done

File: src/css/header.css (hunk 5 above)
Comment (Sysqwerty) [reply src/css/header.css:45]: Done

File: src/css/header.css (hunk 6 above)
Comment (Sysqwerty) [reply src/css/header.css]: Done

**Review by kazamov — APPROVED** [review]

**Pull request merged**